cmd/extract.go

```diff
@@ -37,9 +37,15 @@ func newExtractCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var extractCmd = &cobra.Command{
 		Use:   "extract",
-		Short: "Extract the stack and your Appsody project to a local directory",
-		Long: `This copies the full project, stack plus app, into a local directory
-in preparation to build the final container image.`,
+		Short: "Extract your Appsody project to a local directory.",
+		Long: `Extract the full application (the stack and your Appsody project), into a local directory, in preparation to build the final container image.
+		
+By default, your project gets extracted into your local '$HOME/.appsody/extract' directory, unless specified otherwise.`,
+		Example: `  appsody extract --target-dir $HOME/extract/directory
+  Extracts your project from the container to the local '$HOME/extract/directory' on your local system.
+  
+  appsody extract --name my-container
```

Comment:
I'd say remove this example. The `--name` option will very rarely be used so it will probably just confuse people.